### PR TITLE
Add limited SQL features on TiDB Cloud

### PR DESCRIPTION
**Pull request by sunxiaoguang**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/limited-sql-features.md [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+
+[^7]: Drainer and Pump are not supported on TiDB Cloud.
+
+[^8]: Plugin is not supported on TiDB Serverless.
+
+[^9]: [Time to live (TTL)](/time-to-live.md) is currently unavailable on TiDB Serverless.
+
+[^10]: TiDB Serverless enforces strong password policy.
+
+[^11]: The variable is readonly on TiDB Serverless.
```
Comment (ran-huang) [inline tidb-cloud/limited-sql-features.md:246]:
```suggestion
[^11]: The variable is read-only on TiDB Serverless.
```

File: tidb-cloud/limited-sql-features.md [hunk 2]
```diff
@@ -0,0 +1,246 @@
+---
+title: Limited SQL features on TiDB Cloud
+summary: Learn about the limited SQL features on TiDB Cloud.
+---
+
+# Limited SQL features on TiDB Cloud
+
+TiDB Cloud works with almost all workloads that TiDB supports, but there are feature differences between TiDB Self-Hosted and TiDB Dedicated/Serverless. This document describes the limitations of SQL features on TiDB Cloud. We are constantly filling in the feature gaps between TiDB Self-Hosted and TiDB Dedicated/Serverless. If you require these features or capabilities in the gap, contact us for a feature request.
```
[the remark this answers is not in the document]
Comment (sunxiaoguang) [reply tidb-cloud/limited-sql-features.md:8]: Good point, let's put this link.

**Comment by sunxiaoguang** [comment]
All suggestions and comments were addressed. Let's' close this PR and merge changes to #13970 for easier review.

**Pull request closed without merging**